students/chieu_quach/lesson03/List_lab.py

```diff
@@ -0,0 +1,149 @@
+#  Author      - Chieu Quach
+#  Assignment  - Lesson 3
+#  Exercise    - List Lab
+
+
+def loop_func(lenfruit,typeinput,fruit):
+      
+      
+      i = 0
+      #for i in range(0,lenfruit):
+      while i < lenfruit:
+          
+            # check if type typeinput is an integer
+            if type(typeinput) == int:
+                  if typeinput == i:                       
+                        print(typeinput, fruit[i])
+                        
+            # if type typeinput is "p" then print all fruits that begin with "p"
+            elif typeinput   == "P":
+                  typefruit = fruit[i]
+                  typefruit = typefruit[:1]             
+                  if typefruit == "P":
+                            
+                            print (fruit[i], end = " ")
+                            
+            else: 
+                  print (fruit[i], end = " ")
+            i = i + 1   
+                  
+def series1(fruit): 
+      # Use tools like apppend and insert to add user input
+      # item to fruit table
+      
+      # fruit = ["Apples", "Pears", "Oranges", "Peaches"]
+      print (fruit)
+      # Append fruit to the list
+      response = input("Enter a fruit to be added ")
+      fruit.append(response)
+      lenfruit = len(fruit)
+      loop_func(lenfruit,"append",fruit)
```

Comment:
what is a `loop_func` ? it seems to be doing a lot of things - best to have your function do one thing and be clear on what it's doing.